chalice/app.py

```diff
@@ -17,10 +17,23 @@
 import requests
 from flask import json
 
-
 pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), 'chalicelib'))  # noqa
 sys.path.insert(0, pkg_root)  # noqa
 
+XRAY_TRACE = os.environ.get('XRAY_TRACE', 'false') == 'true'  # noqa
```

Comment:
Can we call this `DSS_XRAY_TRACE` to be consistent with the existing naming convention?

Reply:
Changing the logic to 

`XRAY_TRACE = int(os.environ.get('XRAY_TRACE', '0')) > 0`

would fail if the environment variable was misconfigured and allow us to later define different levels of tracing.

Reply:
Nit: Let's not pretend that this is a constant by all-capsing it.

Reply:
> Changing the logic to
> 
> XRAY_TRACE = int(os.environ.get('XRAY_TRACE', '0')) > 0
>
> would fail if the environment variable was misconfigured and allow us to later define different levels of tracing.

Do do think we will actually have multiple levels of profiling?

Reply:
We might. BTW: `DSS_DEBUG` works the same way: 0 is off, 1 enables debug-level logging for DSS code and 2 does so for all code. The problem I have with `true` and `false` is that I think it is 1) uncommon in the Unix world and 2) ambiguous wrt upper/lower case.